src/ui/components/Timeline/index.tsx

```diff
@@ -73,7 +73,7 @@ class Timeline extends Component<PropsFromRedux, { isDragging: boolean }> {
     const { left, width } = this.$progressBar!.getBoundingClientRect();
     const clickLeft = e.clientX;
 
-    const clickPosition = Math.max((clickLeft - left) / width, 0);
+    const clickPosition = Math.min(1, Math.max((clickLeft - left) / width, 0));
```

Comment:
ooh - just figured out that this will be a number from 0 to 1 based on the position :)

Reply:
The slider is converting x coordinates to a 0-1 value, which then gets converted to a timestamp. The `Math.max` check was there before to prevent the value from dropping below 0 (and so to a negative time). The new `Math.min` check prevents the value from going over 1 (and so to a time that exceeds the Replay profile).